_roadmap/index.md

```diff
@@ -8,10 +8,11 @@ header:
   overlay_image: /assets/images/hosts.jpg
 excerpt: "The long-term roadmap"
 ---
+## The SAMPL Roadmap
 
-**Roadmap**: Details to be filled in here
```

Comment:
Do we want to put the timeline schematic here?

Reply:
![Timeline_new_rough_draft4](https://user-images.githubusercontent.com/29715177/57797375-26ba6a80-76ff-11e9-8826-51723ac3a978.png)

Reply:
I'm debating about that. On the one hand I like the graphic, and on the other it will be a pain to keep it updated (it is already out of date). What I'd like is some sort of graphical table that we can easily update without having to keep dropping out to an image editing software, making updates, then exporting an image and uploading again. However, I do not yet know how to do this. LMK if you have ideas.

Reply:
There's this website (https://www.latex-tables.com/) but it would basically have the same layout as an excel sheet and I'm not sure how to add mini graphics (I'm not familiar with latex) ![Screen Shot 2019-05-15 at 11 53 15 AM](https://user-images.githubusercontent.com/29715177/57801260-34c0b900-7708-11e9-8681-75a44ed6e533.png)

Reply:
You don't need that website to make LaTeX tables. That is one option, and LaTeX tables can include images. BUT... that would require still compiling a document that we then post.

Better would be something that works essentially natively with GitHub/GitHub Pages. 

Markdown tables would be one option (since our pages are written in Markdown), though I don't see that there's option for colored cells in these. https://www.markdownguide.org/extended-syntax/

Other options are probably possible. Maybe ping Slochower on Slack and see if he has any ideas off the top of his head. We just need to know the right tool I think.

Reply:
Ok I'll message him